tests/remote_config/test_remote_configuration.py

```diff
@@ -282,6 +283,53 @@ def validate(data):
         interfaces.library.validate_remote_configuration(validator=validate)
 
 
+@released(cpp="?", dotnet="2.36.0", golang="?", java="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@coverage.basic
+@scenarios.remote_config_mocked_backend_asm_features
+class Test_RemoteConfigurationExtraServices:
+    """Tests that extra services are sent in the RC message"""
+
+    remote_config_is_sent = False
+
+    def setup_tracer_extra_services(self):
+        self.r_outgoing = weblog.get("/createextraservice/?serviceName=extraVegetables")
+
+    def test_tracer_extra_services(self):
```

Comment:
if you do not plan to add more test logic, I suggest using schema, rather than python to do this test